frontend/src/shared/ui/CustomTable/CustomTable.tsx

```diff
@@ -0,0 +1,38 @@
+import { TableContainer, Paper, Table } from "@mui/material";
+import type { FC } from "react";
+import {
+  CustomTableHead,
+  CustomTableRow,
+  CustomTableCell,
+  CustomTableBody,
+} from "./CustomTable.style";
+import { v4 as uuidV4 } from "uuid";
+
+type CustomTableProps = {
+  headerCells: string[];
+  bodyCells: (string | number)[];
+};
+
+export const CustomTable: FC<CustomTableProps> = ({
+  headerCells,
+  bodyCells,
+}) => (
+  <TableContainer component={Paper}>
+    <Table aria-label="simple table">
+      <CustomTableHead>
+        <CustomTableRow>
+          {headerCells.map((item) => (
+            <CustomTableCell key={uuidV4()}>{item}</CustomTableCell>
+          ))}
+        </CustomTableRow>
+      </CustomTableHead>
+      <CustomTableBody>
+        <CustomTableRow>
+          {bodyCells?.map((item) => (
+            <CustomTableCell key={uuidV4()}>{item}</CustomTableCell>
```

Comment:
![image](https://github.com/user-attachments/assets/167a631e-e2f5-4122-8e0e-c2d672947bd4)
https://dev.to/bytebodger/comment/1df8g